IdentityCore/src/validation/MSIDAadAuthorityCache.m

```diff
@@ -75,17 +75,18 @@ + (MSIDAadAuthorityCache *)sharedInstance
 }
 
 - (BOOL)processMetadata:(NSArray<NSDictionary *> *)metadata
+   openIdConfigEndpoint:(NSURL *)openIdConfigEndpoint
               authority:(NSURL *)authority
                 context:(id<MSIDRequestContext>)context
-                  error:(NSError * __autoreleasing *)error
+                  error:(NSError **)error
 {
     if (metadata != nil)
```

Comment:
what if metadata is nil?

Reply:
this scenario is covered by testProcessMetadata_whenNilMetadata_shouldCreateDefaultEntry